tox.ini

```diff
@@ -4,4 +4,16 @@ show-source = True
 max-line-length = 100
 application-import-names = smarkets
 max-cognitive-complexity = 15
-ignore = FI1,I100,W503
+ignore = FI1,W503
+
+[isort]
```

Comment:
It will be convenient to move isort configuration to pyproject.toml since the idea is to move flake8 config also and remove tox.ini

Reply:
Ok